cloud: new generation of TiDB Cloud support services - Connected Care (#19958)
pingcap/docs: tidb-cloud/tidb-cloud-support.md

# TiDB Cloud Support

> **Note:**
>

> > Please note that we have a new announcement about the new generation of TiDB Cloud support services, see [Announcing the new generation of TiDB Cloud support services - Connected Care](/tidb-cloud/connected-care-announcement.md) and [Connected Care Detail](/tidb-cloud/connected-care-detail.md).

Reviewer: ```suggestion
> TiDB Cloud will launch the next-generation of support services, **Connected Care**, on Feb 15, 2025. For more information, see [Announcing the new generation of TiDB Cloud support services - Connected Care](/tidb-cloud/connected-care-announcement.md) and [Connected Care Detail](/tidb-cloud/connected-care-detail.md).
```

> > TiDB Cloud will launch the next-generation of support services, **Connected Care**, on Feb 15, 2025. For more information, see [Announcing the new generation of TiDB Cloud support services - Connected Care](/tidb-cloud/connected-care-announcement.md) and [Connected Care Details](/tidb-cloud/connected-care-detail.md).

Reviewer 2: ```suggestion
> TiDB Cloud will launch the next-generation of support services, **Connected Care**, on Feb 15, 2025. For more information, see [Announcing the New Generation of TiDB Cloud Support Services - Connected Care](/tidb-cloud/connected-care-announcement.md) and [Connected Care Details](/tidb-cloud/connected-care-detail.md).
```

> > TiDB Cloud will launch the next-generation of support services, **Connected Care**, on **Feb 15, 2025**. For more information, see [Announcing the New Generation of TiDB Cloud Support Services - Connected Care](/tidb-cloud/connected-care-announcement.md) and [Connected Care Details](/tidb-cloud/connected-care-detail.md).

Author: ```suggestion
> TiDB Cloud will launch the next-generation of support services, **Connected Care**, on **Feb 17, 2025**. For more information, see [Announcing the New Generation of TiDB Cloud Support Services - Connected Care](/tidb-cloud/connected-care-announcement.md) and [Connected Care Details](/tidb-cloud/connected-care-detail.md).
```
